### PR TITLE
[SURVEY] Commented notification questions

### PR DESCRIPTION
**Pull request by ravenclaw1999**
Commented out notification method and preferences questions. No longer required.

### CODE REVIEW

**Comment by aappacode** [comment]
@ravenclaw1999 can you get the current main changes? i still see the old version 🤔

**Comment by ravenclaw1999** [comment]
> @ravenclaw1999 can you get the current main changes? i still see the old version 🤔

It should be merged now!

File: src/pages/RegisterSurvey.jsx [hunk 1]
```diff
@@ -59,7 +59,7 @@ const RegisterSurvey = () => {
     const handleSubmit = async (e) => {
         e.preventDefault();
 
-        if (!firstName || !lastName || !college || !graduationDate || !monthlyIncome || !moneySaved || !savingsGoal || budgetCategories.length === 0 || !maximumBudget || !phoneNumber || !notificationPreferences || !notificationMethod) {
+        if (!firstName || !lastName || !college || !graduationDate || !monthlyIncome || !moneySaved || !savingsGoal || budgetCategories.length === 0 || !maximumBudget || !phoneNumber) {
```
Comment (aappacode) [inline src/pages/RegisterSurvey.jsx:62]: this will prevent errors, looks good!

File: src/pages/RegisterSurvey.jsx [hunk 2]
```diff
@@ -210,7 +208,7 @@ const RegisterSurvey = () => {
                     className="mt-1 p-2 w-full border rounded-md mb-4"
                 />
                 <br />
-                <label>Notification Preferences</label><br />
+                {/* <label>Notification Preferences</label><br />
```
Comment (aappacode) [inline src/pages/RegisterSurvey.jsx:211]: keeping in case we run into db issues

**Review by aappacode — APPROVED** [review]
added frontend changes. merging now

**Pull request merged**